packages/axios-client/src/index.ts

```diff
@@ -45,7 +45,11 @@ export function getCalculateRetryDelayFn(delayMs: number) {
 }
 
 export function isRequestRetryable(error: AxiosError) {
-  return isNetworkOrIdempotentRequestError(error) || (!error.response && error.code === 'ECONNABORTED')
+  return (
+    isNetworkOrIdempotentRequestError(error) ||
+    (!error.response && error.code === 'ECONNABORTED') ||
+    error.response?.status === 429
```

Comment:
We probably need to have this switched off by default, and enabled only if turned on via a config option, something like: 
```
const client = getAxiosClient({
  retryStatusCodes: [429]`
})
```

Reply:
Ah okay.  Will adjust.